### DISCUSSION TITLE
Is it require my robot connect to the internet ?

### DISCUSSION THREAD

**Discussion by ForteDexe** [Tasshack/dreame-vacuum#208] [General]
I am planning to buy Dreame bot L10s , but I am not sure if I can control it over my local network. Of course it need the internet for initial setup, but I want to disable its internet access later for privacy.

**Comment by Tasshack** [comment]
Internet connection is required if you want the map feature. Also local connection without map support is only available when device is installed to the Mi Home app and not the Dreamehome app.

I do not recommend you to buy a Dreame or any other Xiaomi device if you are planning to use offline.
